web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -1,25 +1,162 @@
 ---
+id: "creating-new-drupal-project"
+title: "Creating a New Drupal Project"
+slug: "/Backend Starters/Decoupled Drupal/Creating a New Project"
 sidebar_position: 2
 ---
 
-# Creating a New Project
+## Choosing an Approach
 
-## Prerequisites
+### Use Build Tools if:
+
+- Testing is an important part of your workflow
+
+- You don’t want to worry about manually pushing changes to your GH repo.
```

Comment:
@lotusbaba  It looks like a lot of these next two sections were copied directly from the 'possible topics' section of the jira ticket. That wasn't intended to be used wholesale and requires a little cleanup to be appropriate to publish.

For example, GH is too informal here.